zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/Collection.java

```diff
@@ -1173,7 +1173,7 @@ public boolean deleteDataVisContent(Session session, Path contentPath) throws IO
             return false;
         }
 
-        String visualisationZipUri = contentPath.toString();
+        String visualisationZipUri = contentPath.getParent().toString();
```

Comment:
This should be fine but it might be wise to check that `getParent()` does not return null or empty. 

We should also check that `visualisationZipUri` is not equal to the `collectionDir` (line 1185).

For example we don't want to delete this dir if its equal to:
- `mycollection/inprogress`
- `mycollection/complete` 
- `mycollection/reviewed`

As this would actually end up deleting the collection directory not just the content - this would be bad. If any of these are true skip the deleted.

Happy to talk through this if you want more of an explanation.